php/Terminus/Loggers/Logger.php

```diff
@@ -55,13 +55,25 @@ public function __construct(
   }
 
   /**
-    * Logs with an arbitrary level.
-    *
-    * @param [mixed]  $level
-    * @param [string] $message
-    * @param [array]  $context
-    * @return [void]
-    */
+   * Logs an error message and then exits with -1
+   *
+   * @param [string] $message Message to convey to user
+   * @param [array]  $context Parts of message to interpolate
+   * @return [void]
+   */
+  public function error($message, array $context = array()) {
+    parent::error($message, $context);
+    exit(-1);
```

Comment:
My thinking at the moment is that the logger probably shouldn't modify the application state (e.g. exit). I think it might be better to have a method named `failure()` that logs an ERROR, then exits.  I think that might be less surprising than having specific log level trigger an exit.

...though this is maybe my obsession with the word "failure", but I think it implies a certain critical behavior than just "error" does.